ansible/manila-client.yml

```diff
@@ -19,9 +20,11 @@
       os_manila_mount_host: "{{ hostvars['localhost']['os_manila_mount_host'] }}"
       os_manila_mount_access_key: "{{ hostvars['localhost']['os_manila_mount_access_key'] }}"
       os_manila_mount_export: "{{ hostvars['localhost']['os_manila_mount_export'] }}"
+      os_manila_mount_share_user: "{{ ceph_mount_share_user }}"
       os_manila_mount_pkgs_install: True
-      os_manila_mount_path: "{{ alaska_homedir }}"
+      os_manila_mount_path: "{{ manila_mount_point }}"
```

Comment:
Also not defined anywhere. Why can't we stick with alaska_homedir?

Reply:
manila_mount_point takes us in the direction of a more general appliances not specific to p3... perhaps a better outcome would be to rename this everywhere?

Reply:
or we do do: os_manila_mount_path: "{{ os_manila_mount_path }}"?

Reply:
The latter would be redundant. I don't think we have a design currently where the playbooks can be reused.